packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -482,44 +475,20 @@ export class PrismicRepositoryManager extends BaseManager {
 			);
 
 			if (error) {
-				return {
-					error: new UnexpectedDataError(
-						`Failed to decode environments: ${error.errors.join(", ")}`,
-					),
-				};
+				throw new UnexpectedDataError(
```

Comment:
Generally, did you test it with an alpha? Because last time it was not possible to see the problem in Dev mode but only with a full SM build :/

Reply:
I did not, but I can try it easily now since we have the new release script. 🙂